<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DreamSculpt</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="gfx.qrc" >:/new/prefix1/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Maximum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="upperFrame" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="bypassButton" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Pass data through without modification</string>
            </property>
            <property name="text" >
             <string>Bypass</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="default" >
             <bool>false</bool>
            </property>
            <property name="flat" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tempoBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Tempo</string>
            </property>
            <property name="minimum" >
             <number>20</number>
            </property>
            <property name="maximum" >
             <number>300</number>
            </property>
            <property name="value" >
             <number>120</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QComboBox" name="scaleRoot" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>Off</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>C</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>C#</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>D</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>D#</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>F</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>F#</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>G#</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>A#</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>B</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="scaleType" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="scaleMode" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <item>
             <property name="text" >
              <string>Up</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Down</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Random</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2" >
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QListWidget" name="arpBox" >
            <property name="layoutDirection" >
             <enum>Qt::LeftToRight</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>9</number>
             </property>
             <property name="topMargin" >
              <number>9</number>
             </property>
             <property name="rightMargin" >
              <number>9</number>
             </property>
             <property name="bottomMargin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QPushButton" name="upButton" >
               <property name="text" >
                <string>Move Up</string>
               </property>
               <property name="icon" >
                <iconset>../../../../usr/share/icons/default.kde/16x16/actions/1uparrow.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="downButton" >
               <property name="text" >
                <string>Move Down</string>
               </property>
               <property name="icon" >
                <iconset>../../../../usr/share/icons/default.kde/16x16/actions/1downarrow.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton" >
               <property name="text" >
                <string>Delete</string>
               </property>
               <property name="icon" >
                <iconset>../../../../usr/share/icons/default.kde/16x16/actions/remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>9</number>
             </property>
             <property name="topMargin" >
              <number>9</number>
             </property>
             <property name="rightMargin" >
              <number>9</number>
             </property>
             <property name="bottomMargin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QPushButton" name="pauseButton" >
               <property name="text" >
                <string>Pause</string>
               </property>
               <property name="checkable" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="noteCombo" >
               <property name="minimum" >
                <number>-24</number>
               </property>
               <property name="maximum" >
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="lengthCombo" >
               <property name="currentIndex" >
                <number>0</number>
               </property>
               <item>
                <property name="text" >
                 <string>2</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/2b.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/semibreve.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/2</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/minim.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/4</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/14.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/8</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/quaver.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/16</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/semiquaver.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/32</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/demisemi.xpm</iconset>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>1/64</string>
                </property>
                <property name="icon" >
                 <iconset resource="gfx.qrc" >:/new/prefix1/gfx/hemidemisemi.xpm</iconset>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addElement" >
               <property name="text" >
                <string>Add</string>
               </property>
               <property name="icon" >
                <iconset>../../../../usr/share/icons/default.kde/16x16/actions/add.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_5" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Recent files&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="recentCombo" />
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHelp_2" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp_2" />
  </widget>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset>../../../../usr/share/icons/default.kde/16x16/actions/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pauseButton</tabstop>
  <tabstop>tempoBox</tabstop>
  <tabstop>arpBox</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>noteCombo</tabstop>
  <tabstop>lengthCombo</tabstop>
  <tabstop>addElement</tabstop>
  <tabstop>recentCombo</tabstop>
  <tabstop>bypassButton</tabstop>
 </tabstops>
 <resources>
  <include location="gfx.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>205</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>noteCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel" >
     <x>123</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
